src/masternodes/rpc_accounts.cpp

```diff
@@ -1993,17 +1993,18 @@ UniValue transferbalance(const JSONRPCRequest& request) {
         targetHeight = ::ChainActive().Height() + 1;
     }
 
-    CTransferBalanceMessage msg;
+    CTransferDomainMessage msg;
 
     try {
         if (!request.params[0].isNull()){
             auto type = request.params[0].getValStr();
-            msg.type = CTransferBalanceType::AccountToAccount;
 
             if (type == "evmin")
-                msg.type = CTransferBalanceType::EvmIn;
+                msg.type = CTransferDomainType::EvmIn;
```

Comment:
- Let's start renaming these to be explicit: `EVMToDVMToken`, `DVMTokenToEVM`. 
- If it makes things, easier, let's just accept enum integer types directly and remove the string repr here. Simpler, easier.